q2_quality_control/plugin_setup.py

```diff
@@ -76,3 +82,87 @@
         'perc_identity==0 and/or perc_query_aligned==0 to disable these '
         'filtering thresholds as necessary.')
 )
+
+plugin.visualizers.register_function(
```

Comment:
Can you stick to one quote mark in the following registration? We bounce back and forth between single and double quotes (I would prefer single quotes - less keystrokes).